samples/matter/lock/sample.yaml

```diff
@@ -29,15 +29,23 @@ tests:
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
     tags: ci_build
-  sample.matter.lock.smp_dfu:
+  sample.matter.lock.smp_dfu_ffs:
     build_only: true
-    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y
+    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y DCONFIG_CHIP_LIB_SHELL=y DCONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y 
+      DCONFIG_CHIP_ROTATING_DEVICE_ID=y DCONFIG_CHIP_DEVICE_TYPE=10
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-      - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
     tags: ci_build
+  sample.matter.lock.smp_dfu_ffs_wifi:
+    build_only: true
+    extra_args: DCONFIG_NET_SHELL=y DCONFIG_NRF700X_REV_A=y DCONFIG_CHIP_DFU_OVER_BT_SMP=y DCONFIG_CHIP_LIB_SHELL=y 
```

Comment:
Do we want to stick with `CONFIG_NRF700X_REV_A=y`? It's enabled for all Matter samples by default. Maybe it would be better to not set it explicitly, as in case of switching to rev B in the future, we may forget to remove it and end up in the situation that twister tests different configuration than our default one.